### DISCUSSION TITLE
Installing old formula version doesn't auto link

### DISCUSSION THREAD

**Discussion by tynanseltzer** [Homebrew/discussions#2590] [Everyday usage] [answered]
Apologies if this has been talked about before and there is a reason for this behavior.

If I do `brew install tbb` it installs and creates the symlink such that tbb is in /usr/local/include.
If I do `brew install tbb@2020` it installs it but does not create the symlink. Is there a reason for this behavior?

**Comment by carlocab** [comment, marked as answer]
Did you read the caveats shown when you installed `tbb@2020`? You can see them again with `brew info tbb@200`.

**Reply by tynanseltzer** [reply]
I did not. Admittedly, I'm guessing very few users do since `brew install foo` tends to generate a lot of text. But point taken, and clearly it's been thought about.

**Reply by carlocab** [reply]
It does generate a lot of text, but it also makes sure to display the caveats last (for possibly a second time depending on whether `brew` needed to install anything else after `tbb@2020`) to make sure you see it.
